### PR TITLE
[m0][mx] Skip queue related tests for m0/mx

### PR DESCRIPTION
**Pull request by yaqiangz**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?
Queue related tests are not expected to run in M0/MX topos

#### How did you do it?
Skip those cases in M0/MX

#### How did you verify/test it?
Run tests

### CODE REVIEW

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -1888,15 +1889,19 @@ telemetry/test_events.py:
 
 telemetry/test_telemetry.py:
   skip:
-    reason: "Skip telemetry test for 201911 and older branches"
+    conditions_logical_operator: or
+    reason: "Skip telemetry test for 201911 and older branches / Unsupported in MGFX topos"
     conditions:
       - "(is_multi_asic==True) and (release in ['201811', '201911'])"
+      - "topo_type in ['m0', 'mx']"
```
Comment (lizhijianrd) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:1896]: Do you aim at skipping entire telemetry test on M0/Mx?

Comment (yaqiangz) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml]: No. Updated

**Review by lizhijianrd — APPROVED** [review]

**Comment by yaqiangz** [comment]
Hi @wangxin could you please help to merge this PR?

**Review by wangxin — APPROVED** [review]

**Comment by yaqiangz** [comment]
Hi @bingwang-ms could you please help to approve the backport request?

**Comment by yaqiangz** [comment]
> Hi @bingwang-ms could you please help to approve the backport request?

No need, there is code conflict, I will manually backport

**Comment by yejianquan** [comment]
Hi @yaqiangz , just FYI, a part of the conditional mark is included in https://github.com/sonic-net/sonic-mgmt/pull/15463, you may need to continue to cherry-pick the remaining parts

**Pull request merged**